src/tutorial/TutorialEventType.cs

```diff
@@ -68,6 +68,12 @@ public enum TutorialEventType
     /// </summary>
     MicrobePlayerReadyToEdit,
 
+    /// <summary>
+    ///   Player presses the button the exit the microbe editor but has made
+    ///   no changes
```

Comment:
```suggestion
    ///   Player presses the button the exit the microbe editor but has made  no changes
```
I'm pretty sure this fits on a single line...